src/utils/getErrors.js

```diff
@@ -0,0 +1,28 @@
+function getErrors(originalText, toCase) {
+  const errors = [];
+
+  if (!originalText.trim()) {
+    errors.push({
+      message: `Text to convert is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+    });
+  }
+
+  if (!toCase) {
+    errors.push({
+      message: `"toCase" query param is required. Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".`,
+    });
+  }
+
+  if (
+    toCase &&
+    !['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'].includes(toCase)
+  ) {
+    errors.push({
+      message: `This case is not supported. Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.`,
```

Comment:
Create a constant for `['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER']` and use it to generate message with available cases